collections/ansible_collections/cloudkit/templates/roles/ocp_4_17_small_github/meta/cloudkit.yaml

```diff
@@ -7,12 +7,12 @@ description: >
 # This sets the nodeRequest field in the ClusterOrder manifest. There must be at
 # least one entry in this list.
 default_node_request:
-- resource_class: fc430
-  number_of_nodes: 2
+- resourceClass: fc430
+  numberOfNodes: 2
 
 # This declares that for a cluster built from this template, only the resource
 # classes listed here may be used when requesting additional nodes. If this
 # list is empty or unspecified, a ClusterOrder may request nodes from any
 # available resource class.
-allowed_resource_classes:
+allowed_resourceClasses:
```

Comment:
super minor note that the casing of this attribute feels weird, with its combination of underscores and camel casing; maybe worth changing now since I don't think this is actually used anywhere?

Reply:
Too much `xargs sed ...`.